pages/events/index.vue

```diff
@@ -77,7 +77,14 @@
         </div>
         <div v-else class="event-page__results">
           <p class="event-page__no-events-msg">
-            Nothing here yet - let’s make it happen!
+            Nothing here yet -
+            <a
+              href="https://airtable.com/shrP84QQiqzC3aWMF"
+              target="_blank"
+              rel="noopener"
```

Comment:
Could you add code to have metrics of how many people click on the link?? For example:
```
<a
  ...
  @click="$trackClickEvent({
       action: 'Qiskit Events: Form to create a new event'
   })"
>
```